dialoguekit/platforms/flask_socket_platform.py

```diff
@@ -84,17 +84,18 @@ def start(self, host: str = "127.0.0.1", port: str = "5000") -> None:
         self.socketio.run(self.app, host=host, port=port)
 
     def display_agent_utterance(
-        self, user_id: str, utterance: Utterance
+        self, agent_id: str, utterance: Utterance, user_id: str
```

Comment:
Can we have user_id as a second argument (so that agent_id and user_id are "grouped" together)?

Reply:
Done.